libraries/libfc/include/fc/log/appender.hpp

```diff
@@ -1,7 +1,7 @@
 #pragma once
 #include <memory>
 
-namespace boost { namespace asio { class io_context; typedef io_context io_service; } }
+namespace boost { namespace asio { class io_context; } }
```

Comment:
What is this for? I feel like this line was originally some sort of boost backwards compatibility thing.

Reply:
Not needed. Removed.